src/components/modules/ContributionsEpic.tsx

```diff
@@ -168,7 +147,18 @@ export const ContributionsEpic: React.FC<EpicProps> = ({
     onReminderOpen,
 }: EpicProps) => {
     const [isReminderActive, setIsReminderActive] = useState(false);
-    const { heading, backgroundImageUrl, showReminderFields, tickerSettings } = variant;
+    const { backgroundImageUrl, showReminderFields, tickerSettings } = variant;
+
+    const cleanHighlighted = replacePlaceholders(variant.highlightedText, numArticles, countryCode);
+    const cleanHeading = replacePlaceholders(variant.heading, numArticles, countryCode);
+
+    const cleanParagraphs = variant.paragraphs.map(paragraph =>
+        replacePlaceholders(paragraph, numArticles, countryCode),
+    );
+
+    if ([cleanHighlighted, cleanHighlighted, ...cleanParagraphs].some(containsPlaceholder)) {
```

Comment:
I noticed `cleanHighlighted` is in here twice, but I think one of them should be `cleanHeading`. I opened #178 to change this.